platforms/android/tangram/src/main/java/com/mapzen/tangram/MapRenderer.java

```diff
@@ -106,17 +102,21 @@ void captureFrame(MapController.FrameCaptureCallback callback, boolean waitForCo
 
     @NonNull
     private Bitmap capture() {
-        View view = map.getGLViewHolder().getView();
+        GLViewHolder viewHolder = map.getGLViewHolder();
+        if (viewHolder == null) {
+            throw new IllegalStateException("MapController GLViewHolder is null");
```

Comment:
Is this possible? How? Out of memory?

Reply:
`getGLViewHolder()` is declared `@Nullable`, which generates a warning unless the result is checked for nullity before accessing it.